win32/src/win32file.i

```diff
@@ -2926,7 +2926,8 @@ static Wow64RevertWow64FsRedirectionfunc pfnWow64RevertWow64FsRedirection = NULL
 
 /* GetFileInformationByHandleEx and supporting structs are defined in SDK for Vista and later,
 	but can also be used on XP with a separate header and lib:
-	http://www.microsoft.com/en-us/download/details.aspx?id=22599
+	https://web.archive.org/web/20121118034013/http://www.microsoft.com:80/en-us/download/details.aspx?id=22599
+    or https://web.archive.org/web/20130718200015/http://www.microsoft.com/en-us/download/confirmation.aspx?id=22599
```

Comment:
I couldn't find a modern equivalent. Although surprisingly this can still be downloaded from the Internet Archive

But I think removing concerns for the EOL Windows XP might be better overall

Reply:
agreed.

Reply:
In this case the code seems like it's still valid. The comment about static linking and different headers just seems irrelevant. Removed !